os/include/sys/mount.h

```diff
@@ -66,6 +66,7 @@
 /****************************************************************************
  * Included Files
  ****************************************************************************/
+#include <tinyara/compiler.h>
```

Comment:
Why do you include this header? Your new codes do not require it.

Reply:
Ok fine